image/image.go

```diff
@@ -65,7 +65,14 @@ func (i *genericImage) Manifest() ([]byte, string, error) {
 			return nil, "", err
 		}
 		i.cachedManifest = m
-		if mt == "" {
+		is_reqmt := false
+		for _, reqmt := range i.requestedManifestMIMETypes {
+			if mt == reqmt {
+				is_reqmt = true
+				break
+			}
+		}
+		if !is_reqmt {
```

Comment:
> I agree to wait after skopeo copy and add that fix as a stopgap for now.

@runcom It is not completely clear which “that fix” you are referring to. The `mt == "text/plain"` special case?

@twaugh FWIW note the test failure, a trivial formal matter really but pointing it out to minimize the number of round trips. This should be possible to reproduce locally by `go build ./... && make .gitvalidation test validate`